node-module-system/calculate.js

```diff
@@ -0,0 +1,28 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+if (process.argv[3] === 'plus') {
```

Comment:
This looks good. It's a bit repetitive, though. While you're waiting for others, think about how you might remove the repetitive code. You could also think about how you could make it easier to add new operators, like "power-of", for example.